python/lsst/ts/m2/tcp_client.py

```diff
@@ -172,6 +175,9 @@ async def _put_read_msg_to_queue(self):
         except json.JSONDecodeError:
             self.log.debug(f"Can not decode the message: {data_decode}.")
 
+        except asyncio.IncompleteReadError:
```

Comment:
If you are just raising the exception again, you probably don't need this block

Reply:
In my test, I need to raise this exception to the upper layer of `try` loop intentionally. Thanks!